src/wp-includes/functions.php

```diff
@@ -4100,10 +4100,10 @@ function _deprecated_hook( $hook, $version, $replacement = null, $message = null
 	if ( WP_DEBUG && apply_filters( 'deprecated_hook_trigger_error', true ) ) {
 		$message = empty( $message ) ? '' : ' ' . $message;
 		if ( ! is_null( $replacement ) ) {
-			/* translators: 1: WordPress hook name, 2: version number, 3: alternative hook name */
+			/* translators: 1: ClassicPress hook name, 2: version number, 3: alternative hook name */
```

Comment:
Let's say "ClassicPress or WordPress hook name" for now.  We'll prefix these version numbers with `WP-` if they are WordPress versions.  See #27.